scraper.py

```diff
@@ -323,10 +323,9 @@ def get_username_from_url(url):
     # Replace username if found in usernames_map
     usernames_map = config.get("usernames_map")
     if usernames_map:
-        username = usernames_map.get(
-            username, # get by the key that should be replaced
-            username) # default to the key if not found
-
+        for key, value in usernames_map:
+            if username == value:
+                return key
```

Comment:
I haven't tested this - but my first impression is that `value` here is of type List. Hence, the comparison should be done differently:

```suggestion
        for key, value in usernames_map:
            if username in value:
                return key
```

Reply:
@elreydetoda can you take a look at this :point_up: 

I might be wrong on this :man_shrugging: